docs/source/getting_started/benchmarks.rst

```diff
@@ -54,6 +55,7 @@ The current benchmark contains the following models:
    "SimMiM", 800, 256, 0.355, "397.8 Min", "10.5 GByte"
    "SimSiam", 800, 256, 0.852, "316.0 Min", "3.9 GByte"
    "SwaV", 800, 256, 0.899, "554.7 Min", "6.6 GByte"
+   "SwaV Queue", 800, 256, 0.894, "273.7 Min", "6.4 GByte"   
```

Comment:
How come it is so much faster than the normal SwaV model? 273 min seems super fast compared to the other models.

Reply:
And do these results already include the gaussian blur fix?

Reply:
It includes the blur fix, that's why it is so high, and I was also surprised by the timing.